src/actions/signInWithCredentials.tsx

```diff
@@ -9,6 +9,7 @@ interface signInWithCredentials {
   history: History<LocationState>;
 }
 
+// eslint-disable-next-line
```

Comment:
you don't need // eslint-disable-next-line
just return this function then change types